ultrasonic/src/main/kotlin/org/moire/ultrasonic/fragment/PlayerFragment.kt

```diff
@@ -979,8 +984,41 @@ class PlayerFragment :
         if (currentPlaying != null) {
             currentSong = currentPlaying!!.song
             songTitleTextView.text = currentSong!!.title
-            albumTextView.text = currentSong!!.album
             artistTextView.text = currentSong!!.artist
+            albumTextView.text = currentSong!!.album
+            if (currentSong!!.year != null && Settings.showNowPlayingDetails)
+                albumTextView.append(String.format(Locale.ROOT, " (%d)", currentSong!!.year))
+
+            genreTextView.text = currentSong!!.genre
```

Comment:
All this block should be inside the `if (Settings.showNowPlayingDetails)`, mostly because if the user is not showing this information it is useless to calculate it.

Reply:
Ok, it is calculated just once per song so it does not really matter. But I changed it. Done.